master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/calicoctl.yaml

```diff
@@ -13,6 +13,7 @@ metadata:
   namespace: kube-system
 spec:
   hostNetwork: true
+  serviceAccountName: calico-node
```

Comment:
While convenient, I think this is the wrong fix.

calico/node and calicoctl have different RBAC requirements, so we shouldn't be sharing a service account for them.

Reply:
To add to that, for a given deployment, there is no guarantee that the `calico-node` svcaccount exists

Reply:
Do we maintain a list of permission calicoctl needs anywhere?

Reply:
@heschlie nope, not in terms of k8s APIs for sure :)

I think you can start with what we have for calico-node, and augment it to include full CRUD of all the resources.

Reply:
yea I started looking at it and will just need to go through and make a list, I figure it doesn't need to watch anything, but it will need update and read on pretty much every resource we care about, and create and delete on several more but not all (like Node, we don't need create and delete correct?)

Reply:
yep, that's right